docs/content/migration-guide/Button.mdx

```diff
@@ -0,0 +1,99 @@
+---
+title: Button migration guide
+---
```

Comment:
@rezrah Should this be in the changelog itself?

Reply:
Yep, we agreed to merge the guides into release notes via changesets

Reply:
@pksjce I can't see a changeset with the release notes at the moment? It's also why CI is failing atm.

Reply:
I added one and then with another commit it totally vanished! I'll add it back now.